source/_components/image_processing.tensorflow.markdown

```diff
@@ -0,0 +1,160 @@
+---
+layout: page
+title: "TensorFlow"
+description: "Detect and recognize objects with TensorFlow."
+date: 2018-10-24 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tensorflow.png
+ha_category: Image Processing
+featured: false
+ha_release: 0.82
+---
+
+The `tensorflow` image processing platform allows you to detect and recognize objects in a camera image using [TensorFlow](https://www.tensorflow.org/). The state of the entity is the number of objects detected, and recognized objects are listed in the `summary` attribute along with quantity. The `matches` attribute provides the confidence `score` for recognition and the bounding `box` of the object for each detection category.
+
+## {% linkable_title Setup %}
+
+TensorFlow object detection requires some dependencies outside of the core tensorflow package.  These live in the [tensorflow/models](https://github.com/tensorflow/models/tree/master/research/object_detection) GitHub repository.  The one we are interested in for this use case lives in `research/object_detection`.  A general outline of the installation process is as follows:
+
+- Clone [tensorflow/models](https://github.com/tensorflow/models/tree/master/research/object_detection)
+- Compile protobuf models located in `research/object_detection/protos` with `protoc`
+- Create the following directory structure inside your config directory:
+
+```bash
+  |- {config_dir}
+    | - tensorflow
+      |- __init__.py
+      |- object_detection/
+```
+
+- Copy required object_detection dependancies to the `object_detection` folder inside of the `tensorflow` folder:
+    - `research/object_detection/data`
+    - `research/object_detection/utils`
+    - `research/object_detection/protos`
+
+A sample script can be found at [this gist](https://gist.github.com/hunterjm/6f9332f92b60c3d5e448ad936d7353c3).
+
+## {% linkable_title Model Selection %}
+
+Finally, you are ready to pick a model.  Researchers have already built multiple TensorFlow object detection models using many open data sets like [COCO](http://cocodataset.org), [Kitti](http://www.cvlibs.net/datasets/kitti/), and [Open Images](https://github.com/openimages/dataset).
+
+This component can support all of them, and if you are feeling adventurous, you can even [roll your own](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/defining_your_own_model.md).
+
+For now, start with one of the COCO models available in the [Model Detection Zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md).  The trade-off here is accuracy vs speed.  The `mAP` is an indication of the model's accuracy while the `ms` is inference speed (on a Nvidia Titan X).  Most users won't see detection speeds close to what is listed.
```

Comment:
Is this too much detail for users? Can we just refer them to the Model Detection Zoo and recommend the `faster_rcnn_inception_v2_coco` or `ssd_mobilenet_v2_coco` models to get started. Then, they can experiment.